web_frontend/osmaxx/excerptexport/models/bounding_geometry.py

```diff
@@ -47,9 +46,9 @@ def __str__(self):
 
 class BBoxBoundingGeometry(BoundingGeometry):
     def __init__(self, *args, **kwargs):
-        attribute_names = ['north', 'east', 'south', 'west']
-        if are_all_keys_in(kwargs, attribute_names) \
-                and not are_all_keys_in(kwargs, ['south_west', 'north_east']):
+        attribute_names = {'north', 'east', 'south', 'west'}
+        if set(attribute_names).issubset(kwargs) \
+                and not {'south_west', 'north_east'}.issubset(kwargs):
```

Comment:
This uses Python builtins whose semantics can be looked up, so we don't have to document them.

While the `are_all_key_in()` function has a very speaking name, the fact that it has to take two arguments makes it not-so-easy (also not incredibly hard, granted) to understand.

If we opt to keep the helper function around, I'd still advice to apply 712ca1ea7e321a8ae68ef7602635c180320dbf36 and d3597857133eb8d988879aab5bdcb1fa705354b3.